markdown/developers/adding_pipelines.md

```diff
@@ -48,50 +48,64 @@ These topics are specifically discussed in the `#new-pipelines` channel:
 ## Create a pipeline from the template
 
 You'll start by making a new pipeline locally and working with it on your own GitHub account.
-Only when it's ready do we move it to the nf-core GitHub organisation.
+Once you have a version of the pipeline that runs, ask the core team to move the repo to the nf-core GitHub organisation for final development, review and release.
```

Comment:
Won't this encourage people to write for the first time in #new-pipelines with fully created pipelines? Which is then difficult to review for the 1st release

Reply:
I'll add a new section for this

Reply:
Added, please check 👍🏻

Reply:
looks good :)